File: years/2022/clojure/aoc-2022/test/aoc_2022/day_09_test.clj
(ns aoc-2022.day-09-test
  (:require [aoc-2022.day-09 :refer :all]
            [clojure.test :refer :all]))

(deftest day-09-test
  (testing "sample data"
    (is (= 13 (part1 "../../data/09-sample.txt"))))
;;     (is (= 8 (part2 "../../data/09-sample.txt"))))
  (testing "puzzle data"
    (is (= 5902 (part1 "../../data/09.txt")))))
;;     (is (= 332640 (part2 "../../data/09.txt")))))

File: years/2018/elixir/test/aoc/day_09_test.exs
defmodule AoC.Day09.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  alias AoC.Day09

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert Day09.part_1(["../data/09.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert Day09.part_2(["../data/09.txt"]) == 1
    end
  end
end
